src/features/advanced/exp-expert-eta.ts

```
@@ -0,0 +1,168 @@
+  if (lines.length === 0) {
+    return undefined;
+  }
+
+  const msToGo = (1 - entry.progress) * expertDays[getTotalExperts(entry)] * MS_IN_DAY;
+  const inProgressOrders = lines
+    .flatMap(line =>
+      line.orders
+        .filter(x => x.completion)
+        .map(x => ({
+          order: x,
+          completion: x.completion!.timestamp,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)